<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="machine_icon">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="machine.qrc">:/machine/icons8-washing-machine-94.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #ffffff;
font-weight: bold;
 margin-bottom: 5px;
font-size: 40px;
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>Yiqa</string>
    </property>
   </widget>
   <widget class="QLabel" name="tagline_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: #ffffff;
    font-weight: normal;
    margin-bottom: 5px;
	 font-size: 20px;
    qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>Welcome to Yiqa â€” Fast. Easy. Reliable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_out">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
border-radius: 6px;
font-size: 14px;
text-decoration: underline

</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QLabel" name="bubbles_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>561</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>bubbles.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>bubbles_label</zorder>
   <zorder>machine_icon</zorder>
   <zorder>name_label</zorder>
   <zorder>tagline_label</zorder>
   <zorder>log_out</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
    <addaction name="action_students"/>
    <addaction name="action_admin"/>
   </widget>
   <addaction name="menuUsers"/>
  </widget>
  <action name="actionAdmin">
   <property name="text">
    <string>Admin</string>
   </property>
  </action>
  <action name="action_students">
   <property name="text">
    <string>Students</string>
   </property>
  </action>
  <action name="action_admin">
   <property name="text">
    <string>Admin</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="machine.qrc"/>
 </resources>
 <connections/>
</ui>
